pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = "pdm.backend"
 
 [project]
 name = "kaiko-eva"
-version = "0.1.0"
+version = "0.1.2"
```

Comment:
should be "0.1.1" first?

Reply:
No, the current version on `pypi` is `0.1.1` already, I'm not sure why in the `.toml` it was still at `0.1.0`